### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by michaeljoelt** [michaeljoelt/tarot#4] [General]
Hey all! Just wanted to get the "discussion" started and test out this functionality! Pretty cool that we can have our own little forums built in to git hub repos.

(if anyone's interested: to enable "Discussions" in your repos, go to Settings -> Features and turn on "Discussions")

Anyway, feel free to introduce yourselves and what your interests are related to coding and/or this project specifically! Thanks!

**Comment by michaeljoelt** [comment]
I guess I'll start- I'm Mike, and my partner got me in to tarot! I am not super familiar with it but have enjoyed using it to help me consider things and brainstorm. I have some pretty sweet decks! I thought this would be a fun app to build and use, and that it would help me learn more about tarot cards and their meanings.

**Comment by EvanderInacio** [comment]
Hi, I'm Evander. I am currently studying javascript.

**Comment by ghost** [comment]
Hi, you can call me Rian. I am currently studying Rust :).

**Comment by michaeljoelt** [comment]
Welcome, Evander and Rian! Coming through from Brazil and Indonesia - that's awesome! 

Feel free to check out or add to current [Issues](https://github.com/michaeljoelt/tarot/issues). I also started a new [discussion for brainstorming ideas](https://github.com/michaeljoelt/tarot/discussions/5). 

Glad to have you here!
